package.json

```diff
@@ -19,25 +19,26 @@
   "author": "Tanel Metsar",
   "license": "MIT",
   "devDependencies": {
-    "@rollup/plugin-alias": "^3.1.8",
-    "@rollup/plugin-node-resolve": "^13.0.6",
-    "@types/jest": "^27.0.3",
-    "@typescript-eslint/eslint-plugin": "^5.4.0",
-    "@typescript-eslint/parser": "^5.4.0",
-    "archiver": "^5.3.0",
-    "bootstrap": "^5.3.0-alpha1",
-    "eslint": "^8.2.0",
-    "fs-extra": "^10.0.0",
-    "glob": "^7.2.0",
-    "jest": "^27.4.4",
-    "rollup": "^2.79.2",
+    "@rollup/plugin-alias": "^5.1.1",
+    "@rollup/plugin-node-resolve": "^16.0.0",
+    "@stylistic/eslint-plugin-ts": "^2.12.1",
+    "@types/jest": "^29.5.14",
+    "archiver": "^7.0.1",
+    "eslint": "^9.17.0",
+    "fs-extra": "^11.2.0",
+    "glob": "^11.0.0",
+    "jest": "^29.7.0",
+    "rollup": "^4.30.1",
     "rollup-plugin-cleanup": "^3.2.1",
     "rollup-plugin-inject-process-env": "^1.3.1",
-    "rollup-plugin-license": "^2.6.0",
-    "rollup-plugin-polyfill": "^3.0.0",
-    "ts-jest": "^27.1.1",
-    "webextension-polyfill": "^0.10.0",
-    "typescript": "^4.5.2"
+    "rollup-plugin-license": "^3.5.3",
+    "rollup-plugin-polyfill": "^4.2.0",
+    "ts-jest": "^29.2.5",
+    "typescript": "^5.7.2",
+    "typescript-eslint": "^8.19.1",
+    "webextension-polyfill": "^0.12.0"
   },
-  "dependencies": {}
+  "dependencies": {
+    "bootstrap": "^5.3.3"
```

Comment:
I think this is not yet needed, this will come with the debug mode.

Reply:
The post-install page uses it: https://github.com/web-eid/web-eid-webextension/blob/main/static/views/installed.html#L6